### DISCUSSION TITLE
mib2 unit suspended

### DISCUSSION THREAD

**Discussion by asdawrwd** [Mr-MIBonk/M.I.B._More-Incredible-Bash#248] [Q&A]
Good morning, my mib2 unit in the 2019 audi s3 has hung up. I connected to it via UART and entered IPL, but I don't know what to do next? I can't find what commands to run.

when i type the command
"boot -t emerg" and immediate mode starts, but there are 60 seconds to enter the command slay -9 MIBEmergency, which I can't add, and the system reboots.

Thank you and please help.

**Comment by harman-f** [comment]
A bit more background would be helpful.
How did the unit get into this state?

You have to login to RCC Emergency 1st, before you can execute `slay -9 MIBEmergency`

**Comment by asdawrwd** [comment]
My unit is MIB Harman. Version MHI2_ER_AU37x_P5089 MU1326.

How to login to RCC Emergency ?

![E0CF3336-468D-4A38-A0DC-694669DE769C](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/133914382/d1f97152-88c9-4b5e-a996-376a61359fda)


after writing the boot -t emerg command and entering emergency ifs, there is a countdown to shutdown the system and at this moment I can't enter anything, neither login nor password because it keeps overwriting the countdown
![0C6FEA76-EBB9-48A8-B0F3-3B12CDA60B11](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/133914382/5a822e37-7b0f-4be3-999c-7ace9f9b314a)

**Reply by harman-f** [reply]
you could write this
![image](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/98130152/f3ee6abe-d59f-4b6e-b088-62bbef9d0a00)

Copy your password to clipboard (strg + c) you can past it into putty with a rigth mouse click.
Password will NOT be shown! not even ***

**Reply by harman-f** [reply]
What got you into this situation?

**Comment by asdawrwd** [comment]
I had a USB socket only for charging, I replaced the USB sockets and the harness. Then I uploaded a patch to unlock the green menu and CarPlay. And then it happened.

are you able to give me some advice?

**Reply by harman-f** [reply]
by patch you mean using M.I.B?

Please share your log file from M.I.B SD card

**Reply by asdawrwd** [reply]
Yes.

where to find this file?

**Comment by harman-f** [comment]
The SD card you used with M.I.B

Folder /backup/ make a zip and upload to e.g. filehorst.de
